docs/hardware/devices/techo/buttons.mdx

```diff
@@ -7,14 +7,14 @@ sidebar_position: 3
 
 ## Functionality
 
-- **Capacitive Touch Button (Top)**
-  - _**Short press**_ updates the e-ink display
-- **Reset Button (Button 1)**
-  - _**Single press**_ resets the device.
-  - _**Double press**_ puts the device into bootloader mode ready to receive new firmware.
-- **Program/Power Button (Button 2)**
-  - _**Single press**_ changes the page displayed on the device.
-  - _**Double press**_ turns the screen backlight on/off
-  - _**Long press**_ signals the device to shutdown.
+- **Capacitive Touch Button (Top):**
+  - **Short press:** Updates the e-ink display.
+- **Reset Button (Button 1):**
+  - **Single press:** Resets the device.
+  - **Double press:** Puts the device into bootloader mode which allows you to update the firmware.
+- **Program/Power Button (Button 2):**
+  - **Single press:** Changes the information page displayed on the device's screen.
+  - **Double press:** Turns the screen backlight on/off.
```

Comment:
From reading the code, but without trying - double pressing will not only toggle the backlight but sends the ad-hoc ping as well.

Reply:
Just checked. It does indeed send (and turn backlight on)

```
INFO  | 21:21:29 31844 [Button] Sending nodeinfo ping to 0xffffffff, wantReplies=1, channel=0
DEBUG | 21:21:29 31844 [Button] cancelSending id=0x38c74cc5, removed=0
INFO  | 21:21:29 31844 [Button] sending owner !7c1afba5/Meshtastic fba5/fba5
DEBUG | 21:21:29 31844 [Button] Update DB node 0x7c1afba5, rx_time=1681420889, channel=0
DEBUG | 21:21:29 31844 [Button] handleReceived(LOCAL) (id=0x38c74cc7 fr=0xa5 to=0xff, WantAck=0, HopLim=3 Ch=0x0 Portnum=4 WANTRESP rxtime=1681420889 priority=10)
DEBUG | 21:21:29 31844 [Button] No modules interested in portnum=4, src=LOCAL
DEBUG | 21:21:29 31844 [Button] localSend to channel 0
DEBUG | 21:21:29 31844 [Button] Add packet record (id=0x38c74cc7 fr=0xa5 to=0xff, WantAck=0, HopLim=3 Ch=0x0 Portnum=4 WANTRESP rxtime=1681420889 priority=10)
DEBUG | 21:21:29 31844 [Button] Expanding short PSK #1
DEBUG | 21:21:29 31844 [Button] Using AES128 key!
DEBUG | 21:21:29 31844 [Button] nRF52 encrypt fr=7c1afba5, num=38c74cc7, numBytes=50!
DEBUG | 21:21:29 31844 [Button] enqueuing for send (id=0x38c74cc7 fr=0xa5 to=0xff, WantAck=0, HopLim=3 Ch=0x8 encrypted rxtime=1681420889 priority=10)
DEBUG | 21:21:29 31844 [Button] txGood=274,rxGood=467,rxBad=0
DEBUG | 21:21:29 31844 [Button] NOTE: tophone queue status queue is full, discarding oldest
DEBUG | 21:21:29 31844 [Screen] Screen: Sent ad-hoc ping
WARN  | 21:21:29 31844 [RadioIf] Can not send yet, busyRx
WARN  | 21:21:29 31844 [RadioIf] Can not send yet, busyRx
WARN  | 21:21:29 31845 [RadioIf] Can not send yet, busyRx
WARN  | 21:21:29 31845 [RadioIf] Can not send yet, busyRx
WARN  | 21:21:29 31845 [RadioIf] Can not send yet, busyRx
WARN  | 21:21:29 31845 [RadioIf] Can not send yet, busyRx
WARN  | 21:21:29 31845 [RadioIf] Can not send yet, busyRx
DEBUG | 21:21:29 31845 [RadioIf] Ignore false preamble detection.
DEBUG | 21:21:30 31845 [RadioIf] Starting low level send (id=0x38c74cc7 fr=0xa5 to=0xff, WantAck=0, HopLim=3 Ch=0x8 encrypted rxtime=1681420889 priority=10)
DEBUG | 21:21:30 31845 [RadioIf] (bw=250, sf=11, cr=4/8) packet symLen=8 ms, payloadSize=66, time 1017 ms
DEBUG | 21:21:30 31845 [RadioIf] AirTime - Packet transmitted : 1017ms
DEBUG | 21:21:31 31846 [RadioIf] Completed sending (id=0x38c74cc7 fr=0xa5 to=0xff, WantAck=0, HopLim=3 Ch=0x8 encrypted rxtime=1681420889 priority=10)
INFO  | 21:21:34 31849 [Button] Sending nodeinfo ping to 0xffffffff, wantReplies=1, channel=0
DEBUG | 21:21:34 31849 [Button] cancelSending id=0x38c74cc7, removed=0
DEBUG | 21:21:34 31849 [Button] Sending NodeInfo will be ignored since we just sent it.
DEBUG | 21:21:34 31849 [Screen] Screen: Sent ad-hoc ping
```